src/plugins/kernel_install/deploy.sh

```diff
@@ -35,6 +35,7 @@ cd "$target_kw_deploy" || exit_msg 'It was not possible to move to deploy dir'
 # Load specific distro script
 . distro_deploy.sh --source-only
 . utils.sh --source-only
+. bootloader.sh --source-only
```

Comment:
Remember that files under `plugins/kernel_install/` are copied to the target machine; in this sense, make sure that we only copy the correct bootloader file.